vuepress/docs/next/tutorials/consume/mt-cds.md

```diff
@@ -0,0 +1,285 @@
+---
+sidebarDepth: 2
+---
+
+# Content Delivery Server for Multitenancy
+You can create a content delivery server (CDS) using descriptors or a custom script to set up the configuration. Then adapt the Entando App Engine to integrate the service for your multitenant application. This tutorial applies descriptors to build the CDS.
+
+## Prerequisites
+* [A working instance of Entando.](../../../docs/getting-started/README.md) with the default Tomcat server image.
+
+## Create the CDS Descriptors
+Descriptors for Keycloak access, ingress, service/deployment, and persistent volume claim are required for the CDS.
+
+### Keycloak Descriptor YAML
+1. Go to Keycloak admin console and log in
+2. From the left sidebar, `Realm Settings` → `Keys`. Click on `Public Key` for `rsa-generated` provider and copy the content.
+
+<details><summary><b>Alternate Method to find the Public Key</b></summary>
+
+1. Retrieve you ingress:
+```
+kubectl get ing | grep default-sso | awk '{print $3} 
+```
+The result of this query will be YOUR-INGRESS
+
+2. Retrieve your public key:
+
+```
+curl "http://YOUR-INGRESS/auth/realms/YOUR-NAMESPACE" | jq '.public_key'
+```
+
+</details>
+
+3. Add the public key to your descriptor file, following the example shown here: 
+
+``` yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: YOUR-KC-SECRET
+  namespace: YOUR-NAMESPACE
+type: Opaque
+stringData:
+  KC_PUBLIC_KEY: "YOUR-PUBLIC-KEY"
+```
+
+### Persistent Volume Claim Descriptor
+This descriptor should provide specifications for the persistent volume claim, including accessModes and resources storage limits. 
+
+``` yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    deployment: YOUR-CDS-DEPLOYMENT
+  name: YOUR-PVC-CLAIM
+  namespace: YOUR-NAMESPACE
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    limits:
+      storage: PVC-SIZE
+    requests:
+      storage: PVC-SIZE
+```
+
+### Service and Deployment Descriptor
+Create the service and deployment descriptor YAML with the specifications required for your environment in a K3s implementation.
+
+``` yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: YOUR-CDS-DEPLOYMENT
+  namespace: YOUR-NAMESPACE
+  labels:
+    app: YOUR-CDS-DEPLOYMENT
+spec:
+  ports:
+    - port: 8080
+      name: internal-port
+      protocol: TCP
+      targetPort: 8080
+    - port: 8081
+      name: public-port
+      protocol: TCP
+      targetPort: 8081
+  selector:
+    app: YOUR-CDS-DEPLOYMENT
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: YOUR-CDS-DEPLOYMENT
+  namespace: YOUR-NAMESPACE
+  labels:
+    app: YOUR-CDS-DEPLOYMENT
+spec:
+  selector:
+    matchLabels:
+      app: YOUR-CDS-DEPLOYMENT
+  template:
+    metadata:
+      labels:
+        app: YOUR-CDS-DEPLOYMENT
+    spec:
+      containers:
```

Comment:
```suggestion
      initContainers:
        - name: init-cds-data
           image: busybox
           imagePullPolicy: IfNotPresent
           command: ['sh', '-c', 'wget --no-check-certificate https://raw.github.com/entando-ps/cds/pr-it-317/entando-data/entando720.tar.gz; tar -xf entando720.tar.gz; rm entando720.tar.gz ']
           volumeMounts:
             - mountPath: /entando-data
                name: cds-data-volume
      containers:
```

Reply:
@avdev4j This doesn't look like the final path since it includes the PR in it - cds/pr-it-317/.  This would be better for now https://github.com/entando-ps/cds/blob/entando720/entando-data/entando720.tar.gz?raw=true but I think we actually need to tag the repo so we can pull a raw file based on the version of Entando.  @pietrangelo what do you suggest here?

Reply:
@nshaw Yes, I used the path given by PJ for testing. since we have the final we can update the doc with cc @jyunmitch